frontend/src/pages/my-projects/components/project-card/project-card.tsx

```diff
@@ -1,17 +1,30 @@
+import { Link } from '~/libs/components/link/link.js';
+import { AppRoute } from '~/libs/enums/app-route.enum.js';
+import { configureString } from '~/libs/helpers/helpers.js';
+import { type ValueOf } from '~/libs/types/types.js';
+
 import styles from './styles.module.scss';
 
 type Properties = {
-  id?: number;
   siteName: string;
   userId?: number;
+  id: number;
```

Comment:
```suggestion
  project: Type
```
pass a full project to props